src/Octoshift/Services/ReclaimService.cs

```diff
@@ -163,6 +161,12 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
                 continue;
             }
 
+            if (lines.Where(x => x.Contains(login)).Count() > 1 && lines.Where(x => x.Contains(userid)).Count() > 1)
+            {
+                _log.LogError($"Mannequin {login} is a duplicate. Skipping.");
```

Comment:
Updated here: cfca6c66dfed923e874e60ba532cf10cf2997463

Kept a couple as "errors" because they were scenarios where the reclamation failed or the reclamation process would not continue due to whatever the error happened to be.